### ISSUE TITLE
app silently closes after aquiring root privileges

### ISSUE THREAD

**Issue by michael-ts** [MarcuzD/TerraInveditDroid#21]
As soon as I tap the "Aquire root" the app quits and I am back to my icons page.  A couple second later I see a toast message indicating that "TeraInveditDroid has been granted" root access for an interactive shell.

Device is Samsung Galaxy Note 10.1 2014 Edition (SM-P600), Android 4.3.

"Mount samespace separation" turned off

Multiuser turned off in your "root app" which is SuperSU 2.46. Also tried updating to latest (2.76)

Latest Busybox (from link) installed.

**Comment by michael-ts** [comment]
I noticed from the source that you are trying to open the directory:

    "/data/data/com.and.games505.TerrariaPaid/files/"

But my app is in:

    "/data/data/com.amz.games505.Terraria/files/"

When I rename the directory it gets past this issue.

Might be nice to present an option to specify where the app is installed.

**Comment by MarcuzD** [comment]
Wait o_O  Original Terraria has multiple names?! Did you got Terraria from the play store? To my knowledge Terraria acquired from Play Store *always* must have the package name "com.and.games505.TerrariaPaid". If I'm right, then it's the Terraria apk's fault, not mine :P

**Comment by michael-ts** [comment]
No, I got it from Amazon's app store, hence the "amz" part, presumably.
